pose-detection/src/blazepose_tfjs/calculators/calculate_world_landmark_projection.ts

```diff
@@ -19,38 +19,33 @@ import {Rect} from '../../calculators/interfaces/shape_interfaces';
 import {Keypoint} from '../../types';
 
 /**
- * Projects normalized landmarks in a rectangle to its original coordinates. The
- * rectangle must also be in normalized coordinates.
- * @param landmarks A normalized Landmark list representing landmarks in a
- *     normalized rectangle.
+ * Projects world landmarks from the rectangle to original coordinates.
+ *
+ * World landmarks are predicted in meters rather than in pixels of the image
+ * and have origin in the middle of the hips rather than in the corner of the
+ * pose image (cropped with given rectangle). Thus only rotation (but not scale
+ * and translation) is applied to the landmarks to transform them back to
+ * original coordinates.
+ * @param landmarks A Landmark list representing world landmarks in the
```

Comment:
I would rather name this parameter "world_landmarks".

Reply:
Fixed in commit Rename variables